Packs/CarbonBlackEnterpriseEDR/pack_metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Carbon_Black_Cloud_Enterprise_EDR",
+    "description": "Advanced threat hunting and incident response solution.",
+    "support": "demisto",
+    "currentVersion": "1.0.0",
+    "author": "Demisto",
+    "url": "",
```

Comment:
need to add our url and email here

Reply:
Our url is: "https://www.paloaltonetworks.com/cortex"?
And what is the email?